src/app/exhibitor/_components/CurrentStatus.tsx

```diff
@@ -17,44 +16,67 @@ export async function CurrentStatus() {
-			<div>
-				<Alert className="mt-0">
-					<Sparkles size={20} />
-					<AlertTitle>Registration is opening soon!</AlertTitle>
-					<AlertDescription>
-						We are preparing the registration for next year&apos;s Armada. In
-						the meanwhile, you are very welcome to report interest in this{" "}
-						<Link
-							className="text-white underline hover:no-underline"
-							href="https://docs.google.com/forms/d/e/1FAIpQLSdny1mhsj1Wutt_FaJtqgxKJP3OOBrWW09Ic3T5_NwEHWhV_w/viewform?usp=sf_link">
-							form
-						</Link>{" "}
-						and we will get back to you once registration is open!
-					</AlertDescription>
-				</Alert>
-			</div>
+			<StatusModuleItem title="Registration is opening soon!">
+				We are preparing the registration for next year&apos;s Armada. In the
+				meanwhile, you are very welcome to report interest in this{" "}
+				<Link
+					className="text-white underline hover:no-underline"
+					href="https://docs.google.com/forms/d/e/1FAIpQLSdny1mhsj1Wutt_FaJtqgxKJP3OOBrWW09Ic3T5_NwEHWhV_w/viewform?usp=sf_link">
+					form
+				</Link>{" "}
+				and we will get back to you once registration is open!
+			</StatusModuleItem>
 		)
 	} else if (
-		new Date(dates.ir.start).getTime() < today &&
+		today > new Date(dates.ir.start).getTime() &&
 		today < new Date(dates.ir.end).getTime()
 	) {
 		return (
-			<div>
-				<Alert className="mt-5">
-					<Sparkles size={20} />
-					<AlertTitle>Initial Registration open</AlertTitle>
-					<AlertDescription>
-						During the Initial Registration you apply to be an exhibitor at
-						Armada. When you do so you commit to exhibit, but you don&apos;t
-						have to specify your package yet. Read more about each stage{" "}
-						<Link
-							className="text-white underline hover:no-underline"
-							href="/exhibitor/timeline">
-							here
-						</Link>
-					</AlertDescription>
-				</Alert>
-			</div>
+			<StatusModuleItem title="Initial Registration open">
+				We are preparing the registration for next year&apos;s Armada. In the
+				meanwhile, you are very welcome to report interest in this During the
+				Initial Registration you apply to be an exhibitor at Armada. When you do
+				so you commit to exhibit, but you don&apos;t have to specify your
+				package yet. Read more about each stage{" "}
+				<Link
+					className="text-white underline hover:no-underline"
+					href="/exhibitor/timeline">
+					here.
+				</Link>{" "}
+			</StatusModuleItem>
+		)
+	} else if (
+		today > new Date(dates.ir.end).getTime() &&
+		today < new Date(dates.fr.start).getTime()
+	) {
+		return (
+			<StatusModuleItem title="Registration Period is over">
+				Registration is over for now, but there might be spots left. Please{" "}
+				<Link
+					className="text-white underline hover:no-underline"
+					href="mailto:sales@armada.nu">
+					contact sales
+				</Link>{" "}
+				if you are interested!
+			</StatusModuleItem>
+		)
+	} else if (
+		new Date(dates.fr.start).getTime() < today &&
```

Comment:
for readability I would reverse this if statement:
`today > new Date(dates.fr.start).getTime()`

Reply:
explanation: Since "today" is always on the left side in all of the other if statements, when you read this line, you have to think backwards

Reply:
reversed for better readability